pandas/io/parquet.py

```diff
@@ -156,12 +162,24 @@ def __init__(self):
             )
         self.api = fastparquet
 
-    def write(self, df, path, compression='snappy', index=None, **kwargs):
+    def write(self, df, path, compression='snappy', index=None,
+              partition_cols=None, **kwargs):
         self.validate_dataframe(df)
         # thriftpy/protocol/compact.py:339:
         # DeprecationWarning: tostring() is deprecated.
         # Use tobytes() instead.
 
+        if 'partition_on' in kwargs:
+            if partition_cols is None:
```

Comment:
Let's simplify this as `if 'partition_on' in kwargs and partition_cols is None`

Reply:
Done